app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,111 @@
+// Place all the styles related to the posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+#sign {
+  font-size: 30px;
+  position: absolute;
+  right: 20px;
+  margin-top: 44px;
+  color: #1D4B60;
+  text-decoration: none;
+  padding-right: 15px;
+  letter-spacing: 1px;
+ }
+
+div a {
+	text-align: center;
+	color: #1D4B60;
+	text-decoration: none;
+	padding: 0px 15px;
+	letter-spacing: 1px;
+}
+
+h3 a {
+	text-align: left;
+	color: #1D4B60;
+	text-decoration: none;
+	padding-left: 0px;
+	margin-bottom:2px;
+	letter-spacing: 1px;
+	font-size: 30px;
+}
+
+h3 {
+	text-align: left;
+	margin: 0px 0px;
+}
+
+img {
+	width: 600px;
+}
+
+#add {
+  width: 300px;
+  margin: auto;
+  margin-top: 30px;
+  font-size: 40px;
+  border: solid #5B5B5E 3px;
+  text-align: center;
+  border-radius: 30px;
+  margin-bottom: 40px;
+
+}
+
+#control {
+	text-align: center;
+}
+
+#post {
+	margin: auto;
+	width: 600px;
+}
+
+#new_post, #edit_post_13, #image {
+	text-align: center;
+}
+
+#image {
+	padding-left: 50px;
+}
+
+#title {
+	font-size: 30px;
+	color: #1D4B60;
+	text-decoration: none;
+	margin-bottom:2px;
+	letter-spacing: 1px;
+	height: 50px;
+}
+
+#post_image {
+	font-size: 10px;
+  color: #1D4B60;
+  margin: 70px 0px;
+  letter-spacing: 1px;
+  transform: scale(1.5);
+  margin-left: 130px;
+}
+
+#post_name {
+	height: 30px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered border-bottom, color, font-size, height, letter-spacing, margin-left, text-decoration, width